hermes-frontend/src/main/java/pl/allegro/tech/hermes/frontend/di/FrontendBinder.java

```diff
@@ -36,6 +38,7 @@ protected void configure() {
 
         bindSingleton(SchemaPrefetchService.class);
         bindSingleton(HealthCheckService.class);
+        bind(HeaderTraceIdExtractor.class).to(TraceExtractor.class).in(Singleton.class);
```

Comment:
could we expose this in `FrontendBuilder` and `ConsumersBuilder`? now you need to know about this secret setting